### PR TITLE
[FRONT-588] Add Testnet 2 config to Network Explorer

### PR DESCRIPTION
**Pull request by juhah**
- Add multiple trackers to `http` config of envs
- Update `streamr-client-protocol` to version `9.0.0`
- Update debug view to render faster by debouncing updates getting rid of coloring the output

### CODE REVIEW

File: src/utils/api/tracker.ts [hunk 1]
```diff
@@ -11,11 +11,11 @@ const getTrackerRegistry = async () => {
...
     return {
-      getAllTrackers: () => [{
-        http: tracker.http,
-      }],
+      getAllTrackers: () => tracker.http.map((url) => ({
+        http: url,
+      })),
       getTracker: () => ({
-        http: tracker.http,
+        http: tracker.http[0],
```
Comment (juhah) [inline src/utils/api/tracker.ts:18]: @hpihkala This logic needs some clarification, how do we select the correct tracker for the stream?

Comment (hpihkala) [reply src/utils/api/tracker.ts:18]: @juhah is this still an open question, or we resolved it in the call?

Comment (juhah) [reply src/utils/api/tracker.ts]: @hpihkala now resolved, switched to using `Utils.createTrackerRegistry` from `streamr-client-protocol`. Testnet 1 config is now:

```js
[{
    id: '0x49D45c17bCA1Caf692001D21c38aDECCB4c08504',
    http: 'https://testnet1.streamr.network:30300',
    ws: 'wss://testnet1.streamr.network:30300',
}]
```

Previously `id` and `ws` were not defined but added for to match `testnet2` config format... Those should be the correct values?

File: src/components/NetworkSelector/index.tsx [hunk 2]
```diff
@@ -40,7 +40,10 @@ const themes: Record<string, Object> = {
   mainnet: {
     color: '#0EAC1B',
   },
-  testnet: {
+  testnet1: {
+    color: '#FF7575',
```
Comment (juhah) [inline src/components/NetworkSelector/index.tsx:44]: Should we use different color for testnet 2?

Comment (hpihkala) [reply src/components/NetworkSelector/index.tsx:44]: Nah that's fine

**Pull request merged**